deepchecks/tabular/checks/data_integrity/mixed_data_types.py

```diff
@@ -79,7 +79,13 @@ def run_logic(self, context: Context, dataset_type: str = 'train') -> CheckResul
             result_dict[column_name] = mix
             if mix:
                 # Format percents for display
-                display_dict[column_name] = {k: format_percent(v) for k, v in mix.items()}
+                formated_mix = {}
+                formated_mix['strings'] = format_percent(mix['strings'])
```

Comment:
If this is for display, than make it prettier, 'Strings', 'Numbers'... also drop the word unique, just "Strings examples"